.node-version

```diff
@@ -1 +1 @@
-4.4.4
+6.6.0
```

Comment:
Cool I just tried it with 4.4.4 and it was fine too. I'll bump it to 6.3.1. Really odd that the build step is consistently not working for handoff in 6.6.0 when you run `make install`